backend/test/test_utils.py

```diff
@@ -0,0 +1,19 @@
+"""utils tests"""
+import pytest
+
+from btrixcloud.utils import slug_from_name
+
+
+@pytest.mark.parametrize(
+    "name,expected_slug",
+    [
+        ("Default org", "default-org"),
+        ("User's org", "users-org"),
+        ("User's @ org", "users-org"),
+        ("Org with åccénted charactêrs", "org-with-accented-characters"),
+        ("Org with åccénted! charactêrs@!", "org-with-accented-characters"),
+        ("cATs! 🐈🐈‍⬛", "cats"),
+    ],
+)
+def test_slug_from_name(name: str, expected_slug: str):
+    assert slug_from_name(name) == expected_slug
```

Comment:
FWIW, this is really just testing the slugify library, right?
Could really just test: `slugify(name.replace("'", ""))` directly.

Reply:
Seems to be working now, so leaving as is!